penguin/web/static/authorize.html

```
@@ -0,0 +1,362 @@
+        <section class="content">
+            <div class="panel" id="status-panel">
+                <p class="status" id="status-text">Paste the startup token printed by <code>penguin-web</code>, or open the full authorization URL from the terminal.</p>
+            </div>
+
+            <div class="panel" id="form-panel">
+                <label class="muted" for="token-input">Startup token</label>
+                <div style="height: 0.5rem"></div>
+                <input id="token-input" class="input" type="password" autocomplete="off" spellcheck="false" placeholder="Paste local startup token" />
+                <div style="height: 1rem"></div>
+                <div class="actions">
+                    <button class="button" id="authorize-button">Authorize browser</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)